dj_cqrs/utils.py

```diff
@@ -0,0 +1,41 @@
+#  Copyright © 2021 Ingram Micro Inc. All rights reserved.
+
+import logging
+from datetime import datetime, timedelta, timezone
+
+from django.conf import settings
+
+
+logger = logging.getLogger('django-cqrs')
+
+
+def get_expires_datetime():
```

Comment:
I don't know what's really better here: to have one result type (return unreachable DT from future) or to do as you've done. What do you think?

Reply:
For me current solution is simplest.

We could add special symbol to describe infinite if you don't like null/None for that. But it will dramatically increase code complexity (message body serialising and deserialising, comparison and etc. ).

Another open question is what is 'unreachable', we don't know at which moment this date will be compared.